From e7299720ceee6cc4a7b10f0a7c168813c08da86c Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 8 Dec 2020 20:22:43 +0100
Subject: [PATCH 1/2] rescue-image: fix compilation

It fixes issue:
cp: cannot stat '/home/beast/beast/workspace/turris-os-packages-dragons-omnia/build/target/linux/mvebu/files-5.4/.': No such file or directory

By looking at OpenWrt repository:
https://github.com/openwrt/openwrt/tree/openwrt-19.07/target/linux/mvebu
I see there is folder files/arch , it is no longer based on kernel patch
version as it was used in OpenWrt 19.07.

For reference:
https://github.com/openwrt/openwrt/tree/openwrt-19.07/target/linux/mvebu
---
 hardware/rescue-image/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hardware/rescue-image/Makefile b/hardware/rescue-image/Makefile
index f174c43c5..f8a9e28cb 100644
--- a/hardware/rescue-image/Makefile
+++ b/hardware/rescue-image/Makefile
@@ -38,7 +38,7 @@ DEVICE := omnia
 else
 DEVICE := unknown
 endif
-kernel_files=$(TOPDIR)/target/linux/$(BOARD)/files-$(KERNEL_PATCHVER)/. $(TOPDIR)/target/linux/generic/files/.
+kernel_files=$(TOPDIR)/target/linux/$(BOARD)/files/. $(TOPDIR)/target/linux/generic/files/.
 
 define Package/rescue-image
 	TITLE:=rescue-image
@@ -98,7 +98,7 @@ define Build/Compile
 		$(STAGING_DIR_HOST)/bin/opkg --offline-root "$(PKG_BUILD_DIR)/rootfs" --force-depends install \{\} \; ;\
 	done
 	rm -rf "$(PKG_BUILD_DIR)/rootfs"/usr/lib/opkg
-	$(KERNEL_MAKE) $(PKG_JOBS) silentoldconfig $(KERNELNAME)
+	$(KERNEL_MAKE) $(PKG_JOBS) oldconfig $(KERNELNAME)
 	cd $(LINUX_DIR); ./scripts/gen_initramfs_list.sh \
 	-o "$(PKG_BUILD_DIR)"/kernel/linux-$(LINUX_VERSION)/arch/$(LINUX_KARCH)/boot/initrd.gz -u squash -g squash \
 	"$(PKG_BUILD_DIR)/rootfs" "$(PKG_BUILD_DIR)/rootfs.extra"
-- 
2.25.1

